docs/source/format/Columnar.rst

```diff
@@ -350,6 +352,53 @@ will be represented as follows: ::
     |----------------|-----------------------|
     | joemark        | unspecified (padding) |
 
+.. versionadded:: Arrow Columnar Format 1.4
+
+Variable-size Binary View Layout
+--------------------------------
```

Comment:
Should move the `versionadded` below the title to avoid any confusion
```suggestion
Variable-size Binary View Layout
--------------------------------

.. versionadded:: Arrow Columnar Format 1.4
```